projects/gdbm/build.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash -eu
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Bootstrap and configure project
+./bootstrap --no-po
+./configure --disable-shared --enable-debug --disable-nls
+# Build project
+make -j$(nproc) all
+# Build fuzzer
+cd fuzz
+$CC $CFLAGS -c -I.. -I../src -ogdbm_fuzzer.o gdbm_fuzzer.c
+$CXX $CFLAGS -ogdbm_fuzzer gdbm_fuzzer.o ../src/libgdbmapp.a ../src/.libs/libgdbm.a $LIB_FUZZING_ENGINE
+
+cp gdbm_fuzzer $OUT
+
+cat > $OUT/gdbm_fuzzer.rc <<EOF
+set errorexit
+open
+set noerrorexit
+set errormask
+avail 
+cache 
+count 
+dir 
+header 
+current 
+first 
+next 
+status 
+set coalesce 
+set centfree 
+recover 
+reorganize 
+fetch 1 
+delete 1 
+store 1 1 
+delete 1 
+bucket 1 
+upgrade 
+downgrade 
+close 
+quit
+EOF
+
+# Create seed
+mkdir seed
+cd seed
+for format in standard numsync; do
```

Comment:
The build script gets a bit bloated IMHO having the seeds like this - would it make sense to make them explicit and place them in the repository here? https://git.gnu.org.ua/gdbm.git/tree/fuzz?h=fuzz